megamek/src/megamek/server/GameManager.java

```diff
@@ -1796,7 +1796,14 @@ private void entityStatusReport() {
             r = new Report(1231);
             r.subject = e.getId();
             r.addDesc(e);
-            r.add(UnitToolTip.getEntityTipReport(e).toString());
+            String etr = "";
+            try {
+                etr = UnitToolTip.getEntityTipReport(e).toString();
+            } catch (Exception ex) {
+                LogManager.getLogger().error("", ex);
+                etr = ex.getMessage();
+            }
+            r.add(etr);
```

Comment:
Generally speaking, we don't stick exception messages into the round reports; logging it is sufficient.